### ISSUE TITLE
Set resource of scanner job in Helm Chart deployment

### ISSUE THREAD

**Issue by PPACI** [artifacthub/hub#1490]
Hello everyone.

I've deployed artifacthub in my company.
I would like to set a resource limit (especially in terms of CPU) for the security scanner job.

I've set the following values to deploy the chart

```yaml
scanner:
  cronjob:
    resources:
      requests:
        memory: 4096Mi
        cpu: 1000m
      limits:
        memory: 4096Mi
        cpu: 1000m
```

But the job was not restricted at all.

When digging the templates, it appears that the resource block is only set to the init container.
Could it also be set for the actual container?

https://github.com/artifacthub/hub/blob/537e4862d0aa0a689eaa66e06354c1fb84eaf134/charts/artifact-hub/templates/scanner_cronjob.yaml#L32

Thank you for your hard work 💪

**Comment by tegioz** [comment]
Hi @PPACI 👋 

This looks like a mistake, the resource block should be set to the `scanner` container. The `tracker` cronjob has the same issue.

Thanks for reporting this, will fix it soon! 👍

**Comment by PPACI** [comment]
Good to hear, thank you!
________________________________

**Comment by tegioz** [comment]
We've just released version `1.1.1` which includes this fix: https://artifacthub.io/packages/helm/artifact-hub/artifact-hub/1.1.1?modal=changelog

Thanks again!
